modules/virt-remove-boot-order-item-web.adoc

```diff
@@ -21,4 +21,12 @@ Remove items from a boot order list by using the web console.
 
 . Click the pencil icon that is located on the right side of *Boot Order*.
 
-. Click the *Remove* icon next to the item. The item is removed from the boot order list and saved in the list of available boot sources. If you remove all items from the boot order list, the following message displays: *No resource selected. VM will attempt to boot disks from YAML by order of appearance in YAML file. Please select a boot source.*
+. Click the *Remove* icon next to the item. The item is removed from the boot order list and saved in the list of available boot sources. If you remove all items from the boot order list, the following message displays: *No resource selected. VM will attempt to boot disks from YAML by order of appearance in YAML file.*
```

Comment:
maybe rephrase the last sentence:
`The VM will attempt to boot the disks from YAML by order of appearance.`

Reply:
Done